tests/examples/awful/template.lua

```diff
@@ -0,0 +1,159 @@
+local file_path, image_path, luacovpath = ...
+
+local cairo      = require("lgi").cairo
+local pango      = require("lgi").Pango
+local pangocairo = require("lgi").PangoCairo
+
+-- Set the global shims
+-- luacheck: globals awesome
```

Comment:
I think it is enough to do `luacheck: globals awesome root tag screen client mouse`, it applies from "here" to the end of the file (unless the comment is on a not-otherwise-empty line)